backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/materialDispo/forecast.feature

```diff
@@ -0,0 +1,92 @@
+@from:cucumber
+@ghActions:run_on_executor6
+Feature: material dispo reacts to forecast docactions
+
+  Background:
+    Given infrastructure and metasfresh are running
+    And the existing user with login 'metasfresh' receives a random a API token for the existing role with name 'WebUI'
+    And metasfresh has date and time 2021-04-11T08:00:00+01:00[Europe/Berlin]
+    And set sys config boolean value true for sys config SKIP_WP_PROCESSOR_FOR_AUTOMATION
+    And AD_Scheduler for classname 'de.metas.material.cockpit.stock.process.MD_Stock_Update_From_M_HUs' is disabled
+
+    And load M_AttributeSet:
+      | M_AttributeSet_ID.Identifier   | Name               |
+      | attributeSet_convenienceSalate | Convenience Salate |
+    And load M_Product_Category:
+      | M_Product_Category_ID.Identifier | Name     | Value    |
+      | standard_category                | Standard | Standard |
+    And update M_Product_Category:
+      | M_Product_Category_ID.Identifier | OPT.M_AttributeSet_ID.Identifier |
+      | standard_category                | attributeSet_convenienceSalate   |
+    And update duration for AD_Workflow nodes
+      | AD_Workflow_ID | Duration |
+      | 540075         | 0        |
+
+  @Id:S0264_200
+  @from:cucumber
+  @Id:S0124_140
+  Scenario: Forecast is correctly considered in Material Dispo when the product is Manufactured
+    Given metasfresh contains M_Products:
+      | Identifier | Name                                | OPT.M_Product_Category_ID.Identifier |
+      | p_1        | trackedProduct_03032025_6           | standard_category                    |
+      | p_2        | trackedProduct_component_03032025_6 | standard_category                    |
+    And metasfresh contains M_PricingSystems
+      | Identifier | Name                           | Value                           | OPT.Description                       | OPT.IsActive |
+      | ps_1       | pricing_system_name_03032025_6 | pricing_system_value_03032025_6 | pricing_system_description_03032025_6 | true         |
+    And metasfresh contains M_PriceLists
+      | Identifier | M_PricingSystem_ID.Identifier | OPT.C_Country.CountryCode | C_Currency.ISO_Code | Name                       | OPT.Description | SOTrx | IsTaxIncluded | PricePrecision | OPT.IsActive |
```

Comment:
pls remove description, sotrx, istaxicluded, priceprecision, isactive columns... they are polluting the test